drivers/ipm/Kconfig.imx

```diff
@@ -64,3 +64,10 @@ config IPM_IMX_MAX_ID_VAL
 	default 1 if IPM_IMX_MAX_DATA_SIZE_8
 	default 0 if IPM_IMX_MAX_DATA_SIZE_16
 	depends on IPM_IMX || IPM_IMX_REV2
+
+config IPM_IMX_FW_READY_REPLY
+	bool "FW_READY reply after boot"
+	default n
```

Comment:
```suggestion
```

Reply:
Why should I remove this?
I don't want this to be enabled for all platforms that are using ipm_imx.

Is it better to use:
`depends on SOC_FAMILY_NXP_ADSP` ?
(https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/xtensa/nxp_adsp/Kconfig#L6 - it refers to the DSP from i.MX targets)

Reply:
`n` is default already. No need to specify it.